pkg/cmd/pulumi/new_test.go

```diff
@@ -818,3 +818,68 @@ func TestErrorIfNotEmptyDirectory(t *testing.T) {
 		})
 	}
 }
+
+func TestProjectExists(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	b := &backend.MockBackend{
+		DoesProjectExistF: func(ctx context.Context, projectName string, orgName string) (bool, error) {
+			type Org string
+			type ProjectName string
+			projects := map[Org]map[ProjectName]struct{}{
+				"moolumi": {
+					"my-moolumi-project": {},
+				},
+				"pulumi": {
+					"my-pulumi-project": {},
+				},
+			}
+			orgProjects, ok := projects[Org(orgName)]
+			if !ok {
+				return false, fmt.Errorf("org %s not found", orgName)
+			}
+			_, exists := orgProjects[ProjectName(projectName)]
+			return exists, nil
+		},
+	}
+
+	// Table Test
+	type Project struct {
+		orgName     string
+		projectName string
+	}
+	tests := []struct {
+		name   string
+		give   Project
```

Comment:
Any reason to declare the separate struct instead of:

```suggestion
	tests := []struct {
		desc   string
		orgName     string
		projectName string
```